cache-config/t3c-apply/torequest/cmd.go

```diff
@@ -85,7 +86,8 @@ func generate(cfg config.Cfg) ([]t3cutil.ATSConfigFile, error) {
 	args = append(args, "--disable-parent-config-comments="+strconv.FormatBool(cfg.DisableParentConfigComments))
 	args = append(args, "--use-strategies="+cfg.UseStrategies.String())
 
-	generatedFiles, stdErr, code := t3cutil.DoInput(configData, config.GenerateCmd, args...)
+	genpath := filepath.Join(t3cutil.InstallDirectory(), config.GenerateCmd)
```

Comment:
So, this is used in a bunch of places, basically every sub-app call.

If you changed them all to use the `t3c` wrapper, e.g. `t3c generate` instead of `t3c-generate`, you could avoid all the duplication, put this in a global var, `var t3cPath = filepath.Join(t3cutil.InstallDir() + "t3c")`, and then all the args would just start with the sub-app, e.g.
```
args := []string{"preprocess"}
args += ...
cmd := exec.Command(t3cPath, args...)
```

Reply:
did as suggested within t3c-apply/torequest/cmd.go